### PR TITLE
Bluetooth: Mesh: Add all missing configuration client model shell command

### PR DESCRIPTION
**Pull request by LingaoM**
Add all missing configuration client model shell command.

Included:

- model publish virtual address set
- model subscription overwrite
- model subscription overwrite virtual address
- model subscription delete all
- low power node poll_timeout get
- key refresh phase get
- node identity get/set
- network key update
- application key update

### CODE REVIEW

File: doc/reference/bluetooth/mesh/shell.rst [hunk 1]
```diff
@@ -528,6 +577,34 @@ The Configuration Client uses the general messages parameters set by ``mesh dst`
 	* ``Model ID``: The model ID of the model to add the subscription to.
 	* ``Company ID``: If present, determines the Company ID of the model. If omitted, the model is a Bluetooth SIG defined model.
 
+``mesh mod-sub-ow <elem addr> <sub addr> <Model ID> [Company ID]``
+-------------------------------------------------------------------
+
+	Overwrite a model from a group address.
```
Comment (trond-snekvik) [inline doc/reference/bluetooth/mesh/shell.rst:583]:
```suggestion
	Overwrite all model subscriptions with a single new group address.
```

File: doc/reference/bluetooth/mesh/shell.rst [hunk 2]
```diff
@@ -528,6 +577,34 @@ The Configuration Client uses the general messages parameters set by ``mesh dst`
...
+
+	* ``elem addr``: Address of the element the model is on.
+	* ``sub addr``: 16-bit group address the model should added to the subscription list (``0xc000`` to ``0xFEFF``).
+	* ``Model ID``: The model ID of the model to add the subscription to.
+	* ``Company ID``: If present, determines the Company ID of the model. If omitted, the model is a Bluetooth SIG defined model.
+
+``mesh mod-sub-ow-va <elem addr> <Label UUID> <Model ID> [Company ID]``
+------------------------------------------------------------------------
+
+	Overwrite the model to a virtual address. Models only receive messages sent to their unicast address or a group or virtual address they subscribe to. Models may subscribe to multiple group and virtual addresses.
```
Comment (trond-snekvik) [inline doc/reference/bluetooth/mesh/shell.rst:593]:
```suggestion
	Overwrite all model subscriptions with a single new virtual address. Models only receive messages sent to their unicast address or a group or virtual address they subscribe to. Models may subscribe to multiple group and virtual addresses.
```

The rest of this line should be included in `mod-sub-ow` as well, or not at all

File: doc/reference/bluetooth/mesh/shell.rst [hunk 3]
```diff
@@ -528,6 +577,34 @@ The Configuration Client uses the general messages parameters set by ``mesh dst`
...
+
+	* ``elem addr``: Address of the element the model is on.
+	* ``Label UUID``: 128-bit label UUID of the virtual address as the new Address to be added to the subscription list. Any omitted bytes will be zero.
+	* ``Model ID``: The model ID of the model to add the subscription to.
+	* ``Company ID``: If present, determines the Company ID of the model. If omitted, the model is a Bluetooth SIG defined model.
+
+``mesh mod-sub-del-all <elem addr> <Model ID> [Company ID]``
+-------------------------------------------------------------------
+
+	Unsubscribe all group address of a model.
```
Comment (trond-snekvik) [inline doc/reference/bluetooth/mesh/shell.rst:603]:
```suggestion
	Remove all group and virtual address subscriptions from a model.
```

File: doc/reference/bluetooth/mesh/shell.rst [hunk 4]
```diff
@@ -539,6 +616,14 @@ The Configuration Client uses the general messages parameters set by ``mesh dst`
 	* ``Company ID``: If present, determines the Company ID of the model. If omitted, the model is a Bluetooth SIG defined model.
 
 
+``mesh krp <NetKeyIdx> [Transition]``
+-------------------------------------
+
+	Get or set the krp value.
```
Comment (trond-snekvik) [inline doc/reference/bluetooth/mesh/shell.rst:622]:
```suggestion
	Get or set the key refresh phase of a subnet.
```

File: doc/reference/bluetooth/mesh/shell.rst [hunk 5]
```diff
@@ -539,6 +616,14 @@ The Configuration Client uses the general messages parameters set by ``mesh dst`
...
+
+	* ``NetKeyIdx``: The identified network key used to Get/Set the current Key Refresh Phase state.
+	* ``Transition``: New Key Refresh Phase Transition.
```
Comment (trond-snekvik) [inline doc/reference/bluetooth/mesh/shell.rst:625]: `phase` instead?

What are the valid values for this?

**Review by trond-snekvik — CHANGES_REQUESTED** [review]
Marking as DNM until #39664 is merged, as this will conflict.

File: doc/reference/bluetooth/mesh/shell.rst (hunk 5 above)
Comment (LingaoM) [reply doc/reference/bluetooth/mesh/shell.rst:625]: 4.3.2.59 Config Key Refresh Phase Set

Comment (trond-snekvik) [reply doc/reference/bluetooth/mesh/shell.rst:625]: Yes, perhaps it could be listed in the docs?

Comment (LingaoM) [reply doc/reference/bluetooth/mesh/shell.rst:625]: How to do ?

Comment (trond-snekvik) [reply doc/reference/bluetooth/mesh/shell.rst:625]:
```suggestion
	* ``Phase``: New Key Refresh Phase. Valid phases are 0, 1 or 2.
```

**Comment by LingaoM** [comment]
@trond-snekvik No conflict, remove DNM now.

**Review by trond-snekvik — APPROVED** [review]

**Comment by LingaoM** [comment]
@carlescufi Can merge this?

**Comment by LingaoM** [comment]
@jhedberg Please take a look?

**Review by jhedberg — APPROVED** [review]

**Pull request merged**